### PR TITLE
Add documentation for amcrest events

### PR DESCRIPTION
**Pull request by FrnchFrgg**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by FrnchFrgg** [comment]
I could not use a language tag because the json language (which is the correct one) treats the payload description as a syntax error (which it is, but json does not have comments, and using a string there may imply the payload is wrapped in a string).

**Comment by MartinHjelmare** [comment]
Please rebase on `next` branch to remove the unrelated commits.

**Comment by FrnchFrgg** [comment]
Oh, I selected the `next` branch in Github's interface but it seems it did not work as I expected. I'll have to fetch and push then (I tried to avoid cloning the repository by working in the web UI)

File: source/_integrations/amcrest.markdown [hunk 1]
```diff
@@ -161,6 +161,23 @@ documentation to install the `ffmpeg`.
 
 Some binary sensors provide two choices for method of operation: streaming or polled. Streaming is more responsive and causes less network traffic because the camera will tell Home Assistant when the sensor's state has changed. Polled mode queries the camera periodically (every five seconds) to check the state of the sensor. Therefore streaming is the better option. However, some camera models and versions of camera firmware do not seem to implement the streaming method properly. Therefore the polled mode is also available. It is recommended to use the streaming mode (e.g., `motion_detected`) first, and if that doesn't work (e.g., results in constant errors), then try the polled mode instead (e.g., `motion_detected_polled`.)
 
+## Events
+
+Once loaded, the `amcrest` integration will generate (home assistant) events when it receives event notifications in the stream sent by the camera. This is only possible if the camera model and firmware implements the streaming method (see [above](#streaming-vs-polled-binary-sensors)). The event type is `amcrest` and the data is as follows:
```
Comment (frenck) [inline source/_integrations/amcrest.markdown]:
```suggestion
Once loaded, the Amcrest integration will generate (Home Assistant) events when it receives event notifications in the stream sent by the camera. This is only possible if the camera model and firmware implement the streaming method (see [above](#streaming-vs-polled-binary-sensors)). The event type is `amcrest` and the data is as follows:
```

**Review by frenck — APPROVED** [review]
Thanks, @FrnchFrgg 👍

**Pull request merged**